auth_oidc_environment/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,14 @@
+To configure this module, you need to:
+
+Create a module server_environment_file with a cfg file or set the environment variable SERVER_ENV_CONFIG with the following section:
+
+[auth_oauth_provider.<name>]
```

Comment:
I always recommend to use tech_name instead which is more reliable

Reply:
Agreed. There is no such field on oauth providers, though.

Reply:
that's why we have the techname.mixin (which I wonder if we shouldn't make the default :thinking: )

Reply:
Ah, interesting. Thanks for the tip!